cellprofiler/modules/relateobjects.py

```diff
@@ -157,9 +162,9 @@ def create_settings(self):
             False,
             doc="""\
 Select "*{YES}*" to calculate the per-parent mean values of every upstream
-measurement made with the children objects and stores them as a
-measurement for the parent; the nomenclature of this new measurements is
-“Mean_<child>_<category>_<feature>”. For this reason, this module
+measurement made with the children objects and store them as a
+measurement for the parent; the nomenclature of this new measurement is
+“Mean_<child>_<category>_<feature>”. This module
 should be placed *after* all **Measure** modules that make measurements
```

Comment:
Could you change "should" to "must" (or even "MUST") to make it more clear that this is mandatory?  This is a confusion point for a large number of users.